packages/Uploader/src/Uploader.js

```diff
@@ -0,0 +1,219 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuidv4 from "uuid/v4";
+import stylers from "@paprika/stylers/";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { inputFileStyles, labelStyles, containerStyles } from "./Uploader.styles";
+import { getFiles } from "./helpers";
+import ProgressBar from "./components/ProgressBar";
+import types from "./types";
+import useDragAndDropEvents from "./useDragAndDropEvents";
+import useProcessFiles from "./useProcessFiles";
+
+const oneMebibyte = 1048576;
+
+const propTypes = {
+  /**
+    Accessible message for the input[type="file"].
+  */
+  a11yText: PropTypes.string,
+  /**
+    An array of string describing the allowed file types for the uploader ex. ["image/*", ".pdf", ".doc", "docx"]
+    reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Unique_file_type_specifiers.
+  */
+  acceptableFileTypes: PropTypes.arrayOf(PropTypes.string),
+  /**
+    When false the uploader only accept one file per upload.
+   */
+  allowMultipleFile: PropTypes.bool,
```

Comment:
`canChooseMultiple`? I'd like to limit how many boolean prefixes we use, if possible.  Need to document it, but `['is', 'are', 'has', 'can', 'should']` should be sufficient.

Here's a recommendation I like that establishes this convention:
https://wiki.eclipse.org/Recommenders/CodingConventions#is_prefix_should_be_used_for_boolean_variables_and_methods.

Reply:
Like their approach.